deploy-kovan-dryrun.json

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
I don't think this file should be in the root dir, as it complicates the mental model.

Right now we can say: we support Kovan and a local testnet, and these are the deploy scripts.

The role of the `dryrun`/`prepare` script is more complicated and not always useful, so I would place it and its files somewhere else.